src/main/java/com/ibm/crypto/plus/provider/AESCipher.java

```diff
@@ -59,7 +59,7 @@ protected byte[] engineDoFinal(byte[] input, int inputOffset, int inputLen)
             byte[] output = null;
             int outputLen = -1;
             if (use_z_fast_command) {
-                output = new byte[engineGetOutputSize2(inputLen)];
+                output = new byte[getOutputSizeForZ(inputLen)];
```

Comment:
Since the `engineGetOutputSize()` now checks `use_z_fast_command`, do we really need the if-else statement here?

Reply:
Yes good point here. I was able to simplify this spot with commit https://github.com/IBM/OpenJCEPlus/pull/263/commits/64d8350ad66b6459c4c9e3629a90b9cf7194c1db since the methods `engineGetOutputSize` and also `engineDoFinal` both handle the logic associated with `use_z_fast_command`.